app/src/main/java/sopt/org/millie/core/designsystem/component/MillieTabbarText.kt

```diff
@@ -0,0 +1,63 @@
+package sopt.org.millie.core.designsystem.component
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.IntrinsicSize
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import sopt.org.millie.core.designsystem.theme.MillieTheme
+
+@Composable
+fun MillieTabbarText(
+    title: String,
+    selected: Boolean = false,
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier,
+
+) {
+    Column (
+        modifier = modifier.width(IntrinsicSize.Max)
+    ) {
+        Text(
+            text = title,
+            modifier = modifier
+                .padding(top = 10.dp)
+                .clickable{onClick()}
+                .padding(bottom = 8.dp),
```

Comment:
```suggestion
                .padding(top = 10.dp, bottom = 8.dp)
                .noRippleClickable(onClick = onClick)
```
패딩 같은 경우는 이렇게 작성해도 똑같은 효과를 볼 수 있고, 우리 리플 효과 지우기 위해서 만든 확장 함수 써주세요 !

Reply:
이부분 이렇게 구현한 이유는 티빙 탑바를 참고했을때 글자 위를 클릭하면 탭바 이동이 안되지만 글자 + 글자와 언더바 사이를 클릭했을때는 이동이 되어서 이렇게 구현했던거에요..! 둘다 가능하게 만들까요? 이거에 대한 제약이 없었어서 티빙 참고했습니다!

Reply:
사용자 관점에서는 글자 위를 클릭했을 때도 탭 전환이 되는 편이 더 자연스러울 것 같아요. 관련 탭을 누르려고 했다면 텍스트뿐 아니라 그 주변 영역까지 클릭할 가능성이 높아서 클릭 영역을 Text가 아닌 Column 전체로 주는 방식이 UX 측면에서 더 적합하다고 생각합니다 !!